src/analyses/local_safe_pointers.cpp

```diff
@@ -237,19 +237,18 @@ void local_safe_pointerst::output_safe_dereferences(
     {
       out << "{";
       bool first = true;
-      for(auto subexpr_it = i_it->code.depth_begin(),
-            subexpr_end = i_it->code.depth_end();
-          subexpr_it != subexpr_end;
-          ++subexpr_it)
-      {
-        if(subexpr_it->id() == ID_dereference)
-        {
-          if(!first)
-            out << ", ";
-          first = true;
-          format_rec(out, subexpr_it->op0());
-        }
-      }
+      i_it->apply([&first, &out](const exprt &e) {
+        for(auto subexpr_it = e.depth_begin(), subexpr_end = e.depth_end();
+            subexpr_it != subexpr_end;
+            ++subexpr_it)
+          if(subexpr_it->id() == ID_dereference)
```

Comment:
This for loop should have braces because its body is spread over multiple lines.